commons/com.b2international.index/src/com/b2international/index/es/EsDocumentSearcher.java

```diff
@@ -518,7 +512,7 @@ private void addSort(DocumentMapping mapping, SearchSourceBuilder reqSource, Sor
 			if (fetchSource) {
 				topHitsAgg
 					.storedFields(STORED_FIELDS_ID_ONLY)
-					.fetchSource(null, EXCLUDED_SOURCE_FIELDS);
```

Comment:
The same thing applies to `TopHitsAggregationBuilder` &ndash; this method is a simple getter.